client/src/app/site/pages/meetings/modules/list-of-speakers-content/modules/point-of-order-dialog/components/point-of-order-dialog/point-of-order-dialog.component.ts

```diff
@@ -50,40 +81,65 @@ export class PointOfOrderDialogComponent {
 
         this.editForm = this.fb.group({
             note: [``, [Validators.maxLength(this.MAX_LENGTH)]],
-            category: []
+            category: [],
+            speaker: []
         });
 
-        combineLatest([
-            this.meetingSettings.get(`list_of_speakers_enable_point_of_order_categories`),
-            this.categoriesSubject
-        ]).subscribe(([enabled, categories]) => {
-            const show = categories.length && enabled;
-            const categoryForm = this.editForm.get(`category`);
-            if (show) {
-                categoryForm.setValidators([Validators.required]);
-                if (!categories.map(cat => cat.id).includes(categoryForm.value)) {
-                    categoryForm.setValue(categories[0].id);
+        this.operator.userObservable.subscribe(user => (this._currentUser = user)),
```

Comment:
```suggestion
        this.operator.userObservable.subscribe(user => (this._currentUser = user));
```
or am I missing something here?